src/app/legacy/psammead/psammead-image/src/index.amp.jsx

```diff
@@ -10,34 +9,16 @@ const omitInvalidProps = omit([
 ]);
 
 const AmpImg = props => {
-  const { srcset, fallbackSrcset, ...otherProps } = props;
+  const { srcset, fallbackSrcset, attribution = '', ...otherProps } = props;
```

Comment:
should the new values be reflected here?

Reply:
Not sure I follow 😬